src/styles/sidebar/components/group-list-item.scss

```diff
@@ -0,0 +1,89 @@
+/* The group. */
+
+.group-list-item {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+
+  .item {
```

Comment:
I forgot to point this out before, but our [CSS style guide](https://github.com/hypothesis/frontend-toolkit/blob/master/docs/css-style-guide.md) recommends avoiding descendant selectors  in most cases (created by nesting in SASS) and instead use `.component__element` names for sub-elements (so. `group-list-item__item` in this case). The advantage is that you can tell more easily what the final styles are on an element just by looking at the element itself, without needing to understand the surrounding context.

In this context, it avoids the possibility that another file could define rules for an `item` class and end up influencing the result here.